python/python/res/server/simulation_context.py

```diff
@@ -175,3 +185,21 @@ def job_status(self, iens):
         except ValueError:
             return None
         return self._queue_manager.getJobStatus(queue_index)
+
+
+    def status_timestamp(self):
+        return self._queue_manager.status_timestamp()
+
+
+    def progress_timestamp(self, iens = None):
+        if iens is None:
+            return self._queue_manager.progress_timestamp(job_index)
+        else:
+            if not iens in self._run_args:
+                raise KeyError("No such simulation: %s" % iens)
+
+            run_arg = self._run_args[iens]
+            queue_index = run_arg.getQueueIndex()
+            return self._queue_manager(queue_index)
```

Comment:
It would be great to have a couple of comments here explaining the difference between `status_timestamp()` and `progress_timestamp()` (called with `iens = None`)

Btw, `job_index` seems to be undefined